observation_portal/observations/filters.py

```diff
@@ -77,4 +77,4 @@ class ConfigurationStatusFilter(django_filters.FilterSet):
 
     class Meta:
         model = ConfigurationStatus
-        fields = '__all__'
+        fields = ('guide_camera_name',)
```

Comment:
what about instrument name?

Reply:
Instrument name is included as a filterable field since it is declared in the filterset a few lines up